HHS/Head-Start-TTADP
# Fix legacy ttaType values

## Description of change

Takes legacy values like empty strings and the word "Both" and converts them to the modern equivalnt

## How to test

Run the migration, then check if `SELECT "ttaType", COUNT(*) FROM "ActivityReports" GROUP BY 1;` returns only the four allowed states e.g.:
```
             ttaType             | count
---------------------------------+-------
 {training,technical-assistance} |  2319
 {technical-assistance}          | 20374
                                 |   516
 {training}                      |  4624
 ```

## Issue(s)

* https://ocio-jira.acf.hhs.gov/browse/TTAHUB-1275


## Checklists

### Every PR


- [ X ] Meets issue criteria
- [ X ] Code is meaningfully tested (run locally)

### Production Deploy


### After merge/deploy
